package dao;

import java.sql.Connection;
import java.sql.PreparedStatement;
import java.sql.SQLException;

import model.User;
import utils.Utils;

public class UserDao {

	public int registerUser(User user) { 

		String INSERT_USER_SQL = "INSERT INTO customer(id, password, name, email, phone, address) "
				+ "VALUES (?, ?, ?, ?, ?, ?);";

		int result = 0;
		
		try {
			Connection conn = Utils.getConnection();
			PreparedStatement pstmt = conn.prepareStatement(INSERT_USER_SQL);

			pstmt.setString(1, user.getId());
			pstmt.setString(2, user.getPassword());
			pstmt.setString(3, user.getName());
			pstmt.setString(4, user.getEmail());
			pstmt.setString(5, user.getPhone());
			pstmt.setString(6, user.getAddress());

			result = pstmt.executeUpdate();

		} catch (SQLException e) {
			System.out.println("SQL 에러");
		}

		return result;
	}

}
